src/vm/amd64/asmhelpers.S

```diff
@@ -39,8 +39,10 @@
 //
 .equ SIZEOF_PROFILE_PLATFORM_SPECIFIC_DATA, 0x8*22 + 0x8  // includes fudge to make FP_SPILL right
 .equ SIZEOF_FP_ARG_SPILL, 0x10*2
+// Buffer to copy structs sequentially
+.equ SIZEOF_PROFILE_STRUCT_BUFFER, 0x8*16
 
-.equ SIZEOF_STACK_FRAME, SIZEOF_PROFILE_PLATFORM_SPECIFIC_DATA + SIZEOF_FP_ARG_SPILL
+.equ SIZEOF_STACK_FRAME, SIZEOF_PROFILE_PLATFORM_SPECIFIC_DATA + SIZEOF_FP_ARG_SPILL + SIZEOF_PROFILE_STRUCT_BUFFER
```

Comment:
Doh! I was thinking about as if it were a member on ProfileArgIterator. I changed it to actually be a member, I think it fits better there since it is just an implementation detail on creating the PROFILE_PLATFORM_SPECIFIC_DATA